swagger.yaml

```diff
@@ -148,6 +148,9 @@ definitions:
       workloads:
         $ref: '#/definitions/workload-list'
         description: List of workloads deployed to the device
+      reconciliation_interval:
```

Comment:
I'm fine with either. Just looking at the code it looked like it is basically a reconciliation loop but being time triggered and not called because of changes in the watched resources. 

How does `workloads_reconciliation_interval` sounds? or the `reconciliation` is not what it's doing in this case (and should stop pushing for it :smile:  ) ? Otherwise `workloads_monitor_interval` is the one that I'm happy with :smile:

Reply:
The primary goal of this go routine was to make sure that any workload that died for any reason is started again

Reply:
> The primary goal of this go routine was to make sure that any workload that died for any reason is started again

Just for died? or more in general `desiredState!=currentState` then take action? 

Apologies for the extra questions :smiley:

Reply:
You're right - it makes sure workloads are in a desired state, but as an emergency and not a primary reconciliation of workloads. That is performed by the `Update` method.

Reply:
just so that I don't forget: minimum value is > 0